docs/configuration.md

```diff
@@ -54,12 +60,31 @@ The structure of a `lighthouserc.json` is segmented by command. Any options you
 }
 ```
 
+##### `lighthouserc.yml`
```

Comment:
similar to the discussion in #125, I'm not sure we need to duplicate *all* of the documentation just to show how yaml works, yaml users should be familiar with how yaml translates to JSON and vice-versa :)

let's just leave a single `lighthouse.yml` example config in the `## Common Examples` section?

Reply:
Sure. I am fine with one example. 

I added the: **YAML Advanced config** at the end. I think it gives a good example about how to use the yaml config file.